roles/ansible/tower/config-ansible-tower-ocp-ldap-ca/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+- include_tasks: ocp_set_ca_certificate.yml
```

Comment:
this should be made conditional - i.e.: only include this task when `ansible-tower.ldap.ca_cert` is defined (and non-emtpy)